packages/storybook/src/stories/grid-filter.stories.tsx

```diff
@@ -322,6 +326,79 @@ export function CommodityWithNewRowsViaApi() {
   );
 }
 
+const useStyles = makeStyles({
+  ratingContainer: {
+    display: 'inline-flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    height: 48,
+    paddingLeft: 20,
+  },
+});
+
+function RatingInputValue(props: FilterInputValueProps) {
+  const classes = useStyles();
+  const { item, applyValue } = props;
+  const [filterValueState, setFilterValueState] = React.useState<number>(Number(item.value));
```

Comment:
The component is controlled, why have a state?

```suggestion
  const filterValueState = Number(item.value);
```